Documentation/README.md

```diff
@@ -24,7 +24,7 @@ We’ve designed the system to make it really easy to share packages on services
 
 ## About Packages
 
-A package consists of Swift source files and a manifest file. The manifest file, called Package.swift, defines the package’s name and its contents using the PackageDescription module. A package has one or more targets. Each target specifies a product and may declare one or more dependencies.
+A package consists of Swift source files and a manifest file. The manifest file, called `Package.swift`, defines the package’s name and its contents using the PackageDescription module. A package has one or more targets. Each target specifies a product and may declare one or more dependencies.
```

Comment:
I find it odd to say that the manifest file is **called** `Package.swift`. Can we change this to. "A package consists of Swift source files, including the `Package.swift` manifest file. The manifest file, or package manifest, defines the package's name..."? This way we mention all different names for the file that we use interchangeably.

Reply:
I agree, it's best to have the various terms for the sake of people doing text searches. I changed to use your proposed phrasing.